bundles/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/handler/CBusTemperatureHandler.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.cbus.handler;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.cbus.CBusBindingConstants;
+
+import com.daveoxley.cbus.Application;
+import com.daveoxley.cbus.CGateException;
+import com.daveoxley.cbus.Group;
+import com.daveoxley.cbus.Network;
+
+/**
+ * The {@link CBusTemperatureHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Scott Linton - Initial contribution
+ */
+@NonNullByDefault
+public class CBusTemperatureHandler extends CBusGroupHandler {
+
+    public CBusTemperatureHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        // Read only thing - no commands to handle
```

Comment:
If this has channels (which I hope it does) then you still need to handle a RefreshCommand.

Reply:
The channels were updated from CBusCGateHandler. now moved into the individual Group Handlers.
I cant get anything to trigger a RefreshCommand but i am not sure when it would be useful. After startup all values are refreshed anyway if possible.

Reply:
The refresh command is issued anytime the UI needs to populate its state information. The easiest way to trigger a RefreshCommand is to just create a new Thing. Also, handling a RefreshCommand is a requirement for all bindings that can support it.